### PR TITLE
RHDEVDOCS-1817 + RHDEVDOCS-2454: Creating a VM via the +Add view of ODC and viewing VMs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.10+

Issue:
- https://issues.redhat.com/browse/RHDEVDOCS-2454
- https://issues.redhat.com/browse/RHDEVDOCS-1817

Link to docs preview:
https://60992--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-create-vms.html#odc-creating-vms_virt-create-vms

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/odc-creating-vms.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* A cluster administrator has installed the {VirtProductName} Operator in your cluster.
+* You have logged in to the {product-title} web console and are in the *Developer* perspective.
+
+.Procedure
+
+. In the *+Add* view, click on the *Virtual Machines* tile to see the *Create new VirtualMachine* page.
+
+. Select a virtual machine type from the *Template catalog*.
+
+. Click *Quick create VirtualMachine* to quick create a virtual machine, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply modules/odc-creating-vms.adoc:22]: Added a note 👍🏻

**Comment by sanketpathak** [comment]
Thanks for the change @abrennan89 
QE approved!

**Review by vikram-raj — COMMENTED** [review]
LGTM

File: modules/odc-creating-vms.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+You can create a virtual machine by using the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* A cluster administrator has installed the {VirtProductName} Operator in your cluster.
+* You have logged in to the {product-title} web console and are in the *Developer* perspective.
+
+.Procedure
+
+. In the *+Add* view, click on the *Virtual Machines* tile to see the *Create new VirtualMachine* page.
```
Comment (bburt-rh) [inline modules/odc-creating-vms.adoc:18]:
```suggestion
. In the *+Add* view, click the *Virtual Machines* tile to see the *Create new VirtualMachine* page.
```
Per ISG, do not use "click on". Use "click" instead.

File: modules/odc-creating-vms.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+. Select a virtual machine type from the *Template catalog*.
+
+. Click *Quick create VirtualMachine* to quick create a virtual machine, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.
```
Comment (bburt-rh) [inline modules/odc-creating-vms.adoc:22]:
```suggestion
. Click *Quick create VirtualMachine* to create a virtual machine with no further input, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.
```
- Not sure if this changes the meaning. Or maybe add `to create a virtual machine immediately`. Anyway, I don't think it should be the exact phrase from the button/link

File: modules/odc-creating-vms.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
++
+[NOTE]
+====
+For Microsoft Windows virtual machines, only the *Customize VirtualMachine* option is available.
+====
+
+. If you choose to customize your virtual machine, the *Customize Template parameters* page allows you to modify *Storage* and *Optional Parameters* settings. When you have finished customizing your virtual machine settings, click *Next* to see the *Review and create VirtualMachine* page. Review your settings, then click *Create VirtualMachine*.
```
Comment (bburt-rh) [inline modules/odc-creating-vms.adoc:29]:
```suggestion
. If you choose to customize your virtual machine, you can modify *Storage* and *Optional Parameters* settings on the *Customize Template parameters* page. When you have finished customizing your virtual machine settings, click *Next* to see the *Review and create VirtualMachine* page. Review your settings, then click *Create VirtualMachine*.
```
- Avoid saying that inanimate things "allow" or "enable" humans to do tasks (anthropomorphism).

**Comment by bburt-rh** [comment]
Lgtm with first and last changes. The middle comment you can take or leave.

File: modules/odc-creating-vms.adoc (hunk 3 above)
Comment (abrennan89) [reply modules/odc-creating-vms.adoc:22]: nice suggestion, thanks

**Pull request merged**